perun-core/src/main/java/cz/metacentrum/perun/core/impl/ResourcesManagerImpl.java

```diff
@@ -530,6 +530,22 @@ public void assignGroupToResource(PerunSession sess, Group group, Resource resou
 		}
 	}
 
+	@Override
+	public void assignGroupToResourceAsInactive(PerunSession sess, Group group, Resource resource) throws GroupAlreadyAssignedException {
```

Comment:
This method is the one where you should be able to specify the status as an argument.

Reply:
Merged to the original `assignGroupToResource` method (impl layer) with status attribute